### PR TITLE
Change 'serial' dual diodes to 'series'

### PR DESCRIPTION
**Pull request by evanshultz**
This is a followup to https://github.com/KiCad/kicad-symbols/pull/2980, which was a followup to https://github.com/KiCad/kicad-symbols/pull/1650. I only changed a dozen dual diode symbols from having `Serial` in the symbol name to `Series`. I made sure to save before making changes and there were none, but unfortunately the Git diff makes things look more confusing than just that simple symbol name change.

@chschlue You merged https://github.com/KiCad/kicad-symbols/pull/2980 so do you mind?

---

## :warning: Deprecation warning
In preparation for the KiCad v6 release this repository will be locked down on Sept 1, 2020. No new pull requests will be accepted. Existing pull requests can be worked on until Oct 1, 2020. Changes breaking v5.1 compatibility can be merged starting Sept 1, 2020.

On Oct 1, 2020 this repository, including issues and pull requests, will be archived and transferred to [gitlab.com](https://gitlab.com/kicad/libraries/kicad-symbols/). In order for your pull requests and issues to be imported into GitLab you must set your email address on GitHub to public and use the same address for your GitLab account. Or login to GitLab at least once using the GitHub icon. Otherwise the importer can't correlate the account information and the issues/comments on GitLab will be owned by `kicad-bot` ([importer documentation](https://docs.gitlab.com/ee/user/project/import/github.html#how-it-works)).

We plan to convert the library to the new v6 S-expr format after it is imported to GitLab. That also means that old pull requests will need to be redone with the v6 format if they are not merged before then.

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [ ] Provide a URL to a datasheet for the symbol(s) you are contributing
- [ ] Provide a screenshot of the symbol(s) from the symbol editor with the pin types visible
- [ ] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

---

Be patient, we maintainers are volunteers with limited time and need to check your contribution against the datasheet. You can speed up the process by providing all the necessary information (see above). And you can speed up the process even more by providing additional info like the screenshot of the symbol editor pin table (or for high pin counts converted to csv) sorted in the same way as the pin table in the datasheet and a direct link to the datasheet page that contains the pin table.

### CODE REVIEW

File: Device.dcm [hunk 1]
```diff
@@ -396,98 +396,98 @@ K diode
...
+$CMP D_Dual_Series_ACK
 D Dual diode, anode/center/cathode
 K diode
 F ~
 $ENDCMP
 #
-$CMP D_Dual_Serial_AKC
-D Dual diode, anode/cathode/center
+$CMP D_Dual_Series_ACK_Parallel
+D Dual diode, common anode on pin 1
```
Comment (chschlue) [inline Device.dcm:406]: No common anode.

File: Device.dcm [hunk 2]
```diff
@@ -396,98 +396,98 @@ K diode
...
 K diode
 F ~
 $ENDCMP
 #
-$CMP D_Dual_Serial_CAK
-D Dual diode, center/anode/cathode
+$CMP D_Dual_Series_ACK_Split
+D Dual diode, common anode on pin 1
```
Comment (chschlue) [inline Device.dcm:412]: No common anode.

File: Device.dcm [hunk 3]
```diff
@@ -396,98 +396,98 @@ K diode
...
+$CMP D_Dual_Series_AKC
+D Dual diode, anode/cathode/center
 K diode
 F ~
 $ENDCMP
 #
-$CMP D_Dual_Serial_KAC
-D Dual diode, cathode/anode/center
+$CMP D_Dual_Series_AKC_Parallel
+D Dual diode, common anode on pin 1
```
Comment (chschlue) [inline Device.dcm:424]: See above

File: Device.dcm [hunk 4]
```diff
@@ -396,98 +396,98 @@ K diode
...
 K diode
 F ~
 $ENDCMP
 #
-$CMP D_Dual_Serial_KCA
-D Dual diode, cathode/center/anode
+$CMP D_Dual_Series_AKC_Split
+D Dual diode, common anode on pin 1
```
Comment (chschlue) [inline Device.dcm:430]: See above.

File: Device.dcm [hunk 5]
```diff
@@ -396,98 +396,98 @@ K diode
...
-D Dual diode, common anode on pin 1
+$CMP D_Dual_Series_CAK
+D Dual diode, center/anode/cathode
 K diode
 F ~
 $ENDCMP
 #
-$CMP D_Dual_Series_ACK_Split
+$CMP D_Dual_Series_CAK_Parallel
 D Dual diode, common anode on pin 1
```
Comment (chschlue) [inline Device.dcm:442]: See above

File: Device.dcm [hunk 6]
```diff
@@ -396,98 +396,98 @@ K diode
...
 K diode
 F ~
 $ENDCMP
 #
-$CMP D_Dual_Series_AKC_Parallel
+$CMP D_Dual_Series_CAK_Split
 D Dual diode, common anode on pin 1
```
Comment (chschlue) [inline Device.dcm:448]: See above

File: Device.dcm [hunk 7]
```diff
@@ -396,98 +396,98 @@ K diode
...
-D Dual diode, common anode on pin 1
+$CMP D_Dual_Series_CKA
+D Dual diode, center/cathode/anode
 K diode
 F ~
 $ENDCMP
 #
-$CMP D_Dual_Series_CAK_Parallel
+$CMP D_Dual_Series_CKA_Parallel
 D Dual diode, common anode on pin 1
```
Comment (chschlue) [inline Device.dcm:460]: See above

File: Device.dcm [hunk 8]
```diff
@@ -396,98 +396,98 @@ K diode
...
 K diode
 F ~
 $ENDCMP
 #
-$CMP D_Dual_Series_CAK_Split
+$CMP D_Dual_Series_CKA_Split
 D Dual diode, common anode on pin 1
```
Comment (chschlue) [inline Device.dcm:466]: See above

File: Device.dcm [hunk 9]
```diff
@@ -396,98 +396,98 @@ K diode
...
-D Dual diode, common anode on pin 1
+$CMP D_Dual_Series_KAC
+D Dual diode, cathode/anode/center
 K diode
 F ~
 $ENDCMP
 #
-$CMP D_Dual_Series_CKA_Split
+$CMP D_Dual_Series_KAC_Parallel
 D Dual diode, common anode on pin 1
```
Comment (chschlue) [inline Device.dcm:478]: See above

File: Device.dcm [hunk 10]
```diff
@@ -396,98 +396,98 @@ K diode
...
 K diode
 F ~
 $ENDCMP
 #
-$CMP D_Dual_Series_KAC_Parallel
+$CMP D_Dual_Series_KAC_Split
 D Dual diode, common anode on pin 1
```
Comment (chschlue) [inline Device.dcm:484]: See above

**Review by chschlue — CHANGES_REQUESTED** [review]
Stopped adding line comments at some point.
Common anode seems to be the common mistake.

**Comment by evanshultz** [comment]
Good catch. I think that should be all of them.

**Review by chschlue — APPROVED** [review]
Thanks.

**Pull request merged**